src/Server/Provision/ServerProvisioner.php

```diff
@@ -171,18 +171,24 @@ private function checkEntities()
     }
 
     /**
-     * @return Entity|void
+     * @return array|void
      */
     public function getEntities()
     {
-        return Entity::query()
+        $ipGroups = $this->ipGroupChoice();
+
+        $query = Entity::query()
             ->where('group', [
-                'billing' => $this->ipGroupChoice(),
+                'billing' => $ipGroups,
             ])
             ->where('billing_id', $this->getIps())
-            ->where('server', 'none')
-            ->first()
-            ;
+            ->where('server', 'none');
+
+        if (count($ipGroups) == 1) {
+            return $query->first();
```

Comment:
in this case we're returning an Entity and in the $query->all() case we're returning an array of Entities. We'll need to keep these consistent. We should probably just do ->all() no matter what.